ConfigSpace/forbidden.pyx

```diff
@@ -218,25 +218,19 @@ cdef class MultipleValueForbiddenClause(AbstractForbiddenClause):
 
 
 cdef class ForbiddenEqualsClause(SingleValueForbiddenClause):
-    """
-    A ForbiddenEqualsClause
+    """A ForbiddenEqualsClause
 
     It forbids a value from the value range of a hyperparameter to be
     *equal to* ``value``.
 
-    Example
-    -------
+    .. code:: python
 
-    >>> cs = CS.ConfigurationSpace()
-    >>> a = CSH.CategoricalHyperparameter('a', [1,2,3])
-    >>> cs.add_hyperparameters([a])
-    [a, Type: Categorical, Choices: {1, 2, 3}, Default: 1]
+        cs = ConfigurationSpace({"a": [1, 2, 3]})
```

Comment:
I really like how much shorter the examples are now.

Reply:
:)